tests/wpt/meta/css/CSS2/floats-clear/__dir__.ini

```diff
@@ -0,0 +1 @@
+prefs: ["layout.tables.enabled:true"]
```

Comment:
Should probably enable the pref too in `css/CSS2/tables/` and `css/css-tables/`. But the latter are currently skipped, so may need to be unskipped first in another PR, to see the difference here?